app/content/team/index.js

```diff
@@ -479,9 +489,7 @@ function Index() {
                           <Icon source="chevron-right" />
                         </View>
                       )}
-                      onPress={() =>
-                        router.push(`content/team/users/${userId}`)
-                      }
+                      onPress={() => handleUserPress(userId)}
```

Comment:
If the function is abstracted it would probably look like this. might be slightly different but just a general idea. 
```suggestion
                      onPress={() => handleUserPress(`content/team/users/${userId}`)}
```

Reply:
Addressed in #296